internal/controller/utils/utils.go

```diff
@@ -536,3 +536,10 @@ func GetGatewayInfoFromConfigMap(ctx context.Context, cli client.Client) (namesp
 
 	return "", "", fmt.Errorf("failed to parse gateway info from configmap")
 }
+
+func HasOpenDataHubManagedLabel(obj client.Object) bool {
+	if labels := obj.GetLabels(); labels != nil {
+		return labels["opendatahub.io/managed"] == "true"
```

Comment:
Some other code is using the standard `app.kubernetes.io/managed-by: odh-model-controller` label. We probably should converge to the standard label.

Example: https://github.com/opendatahub-io/odh-model-controller/blob/8bd306b204a0a3645290bde422927259805d32e4/internal/controller/serving/reconcilers/kserve_istio_smm_reconciler.go#L109-L110.

Reply:
I will follow it up with the following pr